Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static int PromptInt(string message)
+        {
+            Console.Write(message);
+            string input = Console.ReadLine();
+            if (string.IsNullOrEmpty(input)) 
+            {
+                Console.WriteLine();
+                return 0; 
+            }
+            bool isNumber = int.TryParse(input, out int num);
+            Console.WriteLine();
+            if (!isNumber) 
+            {
+                Console.Write($"\"{input}\" is not a valid number.");
+                return -1; 
+            }
+            else if (num == 0) { return 0; }
+            else { return num; }
+        }
+
+        public static string PromptString(string message)
+        {
+            Console.Write(message);
+            string input = Console.ReadLine();
+            bool isString = string.IsNullOrEmpty(input);
```

Comment:
It's not a string, it's an empty string. It's okay to just put the string.IsNullOrempty(input) in the if statement right away.